logging/google/cloud/logging/handlers/_helpers.py

```diff
@@ -36,4 +38,44 @@ def format_stackdriver_json(record, message):
         'severity': record.levelname,
     }
 
+    # Make a best effort to add the trace id.
+    # First try to extract the trace_id from HTTP header, only support flask framework for now.
+    trace_id = parse_flask_request_header()
```

Comment:
Adding Flask-specific, Django-specific, webapp2-specific, pycharm-specific, etc code to support the silver language contract has always been my understanding of the plan.  The exact implementation undetermined, and so now is a good time to talk about it.  Do you have access to the "Stackdriver: Silver Language Requirements" document?  Specifically the "Runtimes: Logging" section.  Shall I setup a meeting to discuss, or take it to email?